packages/dids/src/index.js

```diff
@@ -0,0 +1,34 @@
+import {v4 as uuidv4} from 'uuid';
+const didKeyDriver = require('@digitalbazaar/did-method-key').driver();
+
+const DID_DEFAULT_CONTEXT = [
+  'https://w3id.org/wallet/v1',
+  'https://w3id.org/did-resolution/v1',
+];
+export const DIDKeyManager = (function () {
+  const createDID = async keyDoc => {
+    const {didDocument} = await didKeyDriver._keyPairToDidDocument({
+      keyPair: {
+        ...keyDoc,
+        keyPair: keyDoc,
+      },
+    });
+
+    const expiryDate = new Date();
+    expiryDate.setFullYear(expiryDate.getFullYear() + 1000);
+    const didResolution = {
```

Comment:
to me it makes more sense for this method to return the document, and you can export another method to get a did resolution document from any did document (future proofing)